tool/lint.sh

```diff
@@ -15,3 +15,17 @@ if [ -n "$bad" ]; then
 	echo "$bad"
 	exit 1
 fi
+
+bad=$(grep -Pzo '\)\s*\n\s*\{' src/main/c/cext/ruby.c || exit 0)
```

Comment:
I haven't looked, but are those GNU or POSIX flags? In particular I'm wondering if this works fine out of the box with macOS X or Solaris.